docs/installation-docker.rst

```diff
@@ -0,0 +1,81 @@
+=======================
+Installation via Docker
+=======================
+
+Kuma has experimental support for `Docker`_ as an alternative to the
+Vagrant set up.
+
+.. _Docker: https://www.docker.com/
+
+Docker setup
+============
+
+#. Ensure you have cloned the kuma Git repository and it is your current
+   working directory::
+
+        git clone git@github.com:mozilla/kuma.git
+        cd kuma
+
+#. Install the `Docker Toolbox`_. This will install the Docker engine,
+   along with Docker Compose and Docker Machine.
+
+   .. _Docker Toolbox: https://www.docker.com/products/docker-toolbox
+
+#. Create the virtual machine that Docker will run within::
+
+        docker-machine create --driver virtualbox --virtualbox-memory 2048 kuma
+
+#. Configure the environment variables::
+
+        eval "$(docker-machine env kuma)"
+        export COMPOSE_FILE=`pwd`/docker/docker-compose.yml
+        export COMPOSE_PROJECT_NAME=kuma  # String prepended to every container.
+
+#. Add the Docker VM to your hosts::
+
+        sudo sh -c "echo $(docker-machine ip kuma 2>/dev/null)  kuma.dev >> /etc/hosts"
```

Comment:
It looks like the `--add-host` option adds the host to the container, not the host machine, which is what I'm doing here.

I chose `kuma.dev` to not stomp on the hostname that Vagrant is using, at least the Docker set up is the preferred way to go.  Also, you've reminded me that I haven't set up SSL yet.